lib/travis/build/script/r.rb

```diff
@@ -322,10 +324,13 @@ def r_install(packages)
         def r_github_install(packages)
           return if packages.empty?
           packages = Array(packages)
-          setup_devtools
+
+          setup_remotes
+          setup_devtools if config[:use_devtools]
```

Comment:
What is the purpose of this new configuration parameter?

Reply:
To allow people to fall back to the previous behavior if they desire.

We lose a little functionality using only the remote package. No printing of the installed packages and reprinting of the checks that failed.